includes/wikia/services/ArticleQualityService.php

```diff
@@ -158,6 +160,18 @@ public function getArticleQuality() {
 			$article = new Article( $title );
 			$parser = $article->getParserOutput();
 
+			if ( !$parser ) {
```

Comment:
maybe we could extract this functionallity to seperate method and use type hinting?